samples/crypto/psa_tls/sample.yaml

```diff
@@ -81,6 +81,43 @@ tests:
       - nrf9151dk/nrf9151
       - nrf9151dk/nrf9151/ns
     tags: ci_build cc3xx_oberon dtls sysbuild ci_samples_crypto
+  sample.psa_tls.tls_server.ecdsa.cc3xx_oberon_13:
+    sysbuild: true
+    build_only: true
+    extra_args: >
+      OVERLAY_CONFIG="overlays/server.conf;overlays/ecdsa.conf;overlays/cc3xx-oberon-psa.conf;overlays/tls_13.conf"
+    platform_allow: >
+      nrf5340dk/nrf5340/cpuapp nrf9160dk/nrf9160 nrf5340dk/nrf5340/cpuapp/ns
+      nrf9160dk/nrf9160/ns nrf9161dk/nrf9161 nrf9161dk/nrf9161/ns nrf9151dk/nrf9151
+      nrf9151dk/nrf9151/ns
+    integration_platforms:
+      - nrf5340dk/nrf5340/cpuapp
+      - nrf5340dk/nrf5340/cpuapp/ns
+      - nrf9160dk/nrf9160
+      - nrf9160dk/nrf9160/ns
+      - nrf9161dk/nrf9161
+      - nrf9161dk/nrf9161/ns
+      - nrf9151dk/nrf9151
+      - nrf9151dk/nrf9151/ns
+    tags: ci_build cc3xx_oberon sysbuild ci_samples_crypto
+  sample.psa_tls.tls_client.ecdsa.cc3xx_oberon_13:
```

Comment:
I think `cc3xx_oberon_13`is inconsistent with the other sample names. Maybe something like `sample.psa_tls.tls_1_3.tls_client.ecdsa.cc3xx_oberon`?

Reply:
Yes, I was going to suggest the same

Reply:
Makes more sense to have `tls_1_3` instead of `backend_13` also, so I'll change that